socialhome/streams/app/components/ReactionsBar.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="grid-item-bar d-flex justify-content-start">
+        <slot />
```

Comment:
What is this? Does it receive the stuff passed into as content when calling this component?

Reply:
Yes. It's mainly to keep the current design while leaving the timestamp, delete and edit links, outside the component. It's not pretty, but I think this will change when I start to work on the design.